scripts/dist-tag.ts

```diff
@@ -0,0 +1,51 @@
+/**
+ * @license
+ * Copyright Google Inc. All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+// tslint:disable: no-implicit-dependencies
+
+import { logging } from '@angular-devkit/core';
+import { execSync } from 'child_process';
+import { packages, stableToExperimentalVersion } from '../lib/packages';
+
+interface DistTagOptions {
+  /**
+   * Version must be specified in format d+.d+.d+ where d is a 0-9 digit.
+   * This must be a stable version with major version > 0.
+   * The script will automatically convert stable version to experimental.
+   */
+  version: string;
+  /**
+   * Tag is usually "latest" or "next", but could also be "v10-lts" for example.
+   */
+  tag: string;
+}
+
+/**
+ * This function adds a tag to all public packages in the CLI repo.
+ */
+export default function(args: DistTagOptions, logger: logging.Logger) {
```

Comment:
Nit: I'm not too familiar with the framework, but is there any help or explanation content that would display for this subcommand?

Can we do `yarn admin dist-tag --help` or `yarn admin help dist-tag` and get anything useful? If that's not supported then consider adding `--help` as an argument. It isn't too big a deal, but if this is easy then we should make sure to add the relevant metadata to support it.

Reply:
The "framework" aka devkit-admin doesn't really support help, since it's mostly a loose collection of scripts.
I've added `--help` as a command line argument since that's the easiest.